go.mod

```diff
@@ -1,14 +1,14 @@
 module github.com/elastic/fleet-server/v7
 
-go 1.24
+go 1.24.1
```

Comment:
This changed because github.com/elastic/elastic-agent-libs@v0.19.0 has a `go 1.24.1` directive in it's `go.mod` file.